### PR TITLE
Add http-server.https.[included/excluded]-cipher configuration

### PR DESCRIPTION
**Pull request by haozhun**
@dain

### CODE REVIEW

**Comment by haozhun** [comment]
Addressed comment from @martint

File: http-server/src/main/java/io/airlift/http/server/HttpServer.java [hunk 1]
```diff
@@ -169,6 +170,14 @@ public HttpServer(HttpServerInfo httpServerInfo,
 
             SslContextFactory sslContextFactory = new SslContextFactory(config.getKeystorePath());
             sslContextFactory.setKeyStorePassword(config.getKeystorePassword());
+            String includedCipherSuites = config.getHttpsIncludedCipherSuites();
+            if (!isNullOrEmpty(includedCipherSuites)) {
+                sslContextFactory.setIncludeCipherSuites(includedCipherSuites.split(","));
```
Comment (electrum) [inline http-server/src/main/java/io/airlift/http/server/HttpServer.java]: This splitting should be done in the config class

Comment (dain) [inline http-server/src/main/java/io/airlift/http/server/HttpServer.java]: Use `Splitter.on(',').trimResults().omitEmptyStrings().splitToList()`

File: http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java [hunk 2]
```diff
@@ -162,6 +169,30 @@ public HttpServerConfig setKeystorePassword(String keystorePassword)
...
+
+    public List<String> getHttpsExcludedCipherSuites()
+    {
+        return excludedCipherSuites;
+    }
+
+    @Config("http-server.https.excluded-cipher")
+    public HttpServerConfig setHttpsExcludedCipherSuites(String excludedCipherSuites)
+    {
+        this.excludedCipherSuites = Splitter.on(',').trimResults().omitEmptyStrings().splitToList(requireNonNull(excludedCipherSuites, "excludedCipherSuites is null"));
```
Comment (dain) [inline http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java]: one per line

**Comment by dain** [comment]
looks good

**Comment by dain** [comment]
Looks good

**Pull request merged**